views/base.html.twig

```diff
@@ -86,16 +86,15 @@
                             </li>
                         {% endif %}
 
-                        {% if path_exists('help_home') %}
-                            <li class="dropdown">
-                                <a href="{{ path('help_home') }}" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><i class="fa fa-question-circle" aria-hidden="true"></i> Help <span class="caret"></span></a>
-                                <ul class="dropdown-menu">
-                                    <li><a href="{{ path('help_videos') }}"><i class="fa fa-film" aria-hidden="true"></i> Training Videos</a></li>
-                                    <li><a href="{{ path('help_faq') }}"><i class="fa fa-question-circle-o" aria-hidden="true"></i> Technical FAQs</a></li>
-                                    <li><a href="{{ path('help_sop') }}"><i class="fa fa-file-text-o" aria-hidden="true"></i> All of Us℠ SOPs</a></li>
-                                </ul>
-                            </li>
-                        {% endif %}
+                        <li class="dropdown">
+                            <a href="/s/help/" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><i class="fa fa-question-circle" aria-hidden="true"></i> Help <span class="caret"></span></a>
+                            <ul class="dropdown-menu">
+                                <li><a href="/s/help/videos/"><i class="fa fa-film" aria-hidden="true"></i> Training Videos</a></li>
+                                <li><a href="/s/help/faq/"><i class="fa fa-question-circle-o" aria-hidden="true"></i> Technical FAQs</a></li>
+                                <li><a href="/s/help/sop/"><i class="fa fa-file-text-o" aria-hidden="true"></i> All of Us℠ SOPs</a></li>
+                            </ul>
+                        </li>
```

Comment:
I keep waffling between wanting to put these routes in the Silex [`SymfonyMigrationController`](https://github.com/all-of-us/healthpro/blob/develop/src/Pmi/Controller/SymfonyMigrationController.php) or hard-linking them as done here and not worrying about that.

**Pros:**
- Reduces the number of templates we have to change / search for
- Conceivably offers a spot to change the request logic without mass-refactoring (not relevant here)

**Cons:**
- User's browser makes an extra request to bounce to the new section
- Extra code to maintain later (until the migration is complete)